CHANGELOG.md

```diff
@@ -59,6 +59,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
   * CLI: removed `--text` flag from `show-node-id` command; the text format for public keys is not used any more - instead we use ProtoJSON.
 * (types) [\#9079](https://github.com/cosmos/cosmos-sdk/issues/9079) Add `AddAmount`/`SubAmount` methods to `sdk.Coin`.
 * [\#8628](https://github.com/cosmos/cosmos-sdk/issues/8628) Commands no longer print outputs using `stderr` by default
+* [\#9134](https://github.com/cosmos/cosmos-sdk/pull/9134) Renamed the CLI flag `--memo` to `--note`.
```

Comment:
is this the only reference we have/need to change? I found many other references for Memo on the code other than the flag

Reply:
if not it's ok but I'm just confused by the title of the PR

Reply:
see https://github.com/cosmos/cosmos-sdk/issues/9122#issuecomment-821722875